openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -272,6 +275,17 @@ jQuery(function () {
             });
     }
 
+    const $shareModalLinks = $('.share-modal-link');
+    if ($shareModalLinks.length) {
+        import('./Share')
+        import(/* webpackChunkName: "modal-links" */ './modals')
+            .then(module => {
+                module.initShareModal($shareModalLinks)
+            });
```

Comment:
```suggestion
        import(/* webpackChunkName: "share-links" */ './Share')
            .then(module => {
                module.initShareModal($shareModalLinks)
            });
```
Let's import from `Share.js`.